src/backend/AsyncMirror.ts

```diff
@@ -106,7 +107,7 @@ export default class AsyncMirror extends file_system.SynchronousFileSystem imple
                 if (err) {
                   cb(err);
                 } else if (i < files.length) {
-                  copyItem(`${p}/${files[i]}`, copyNextFile);
```

Comment:
Did this line cause an error at one point? (I avoided `join` because it's more expensive)